client/store/modules/activeUsers/mutations.js

```diff
@@ -0,0 +1,75 @@
+import find from 'lodash/find';
+import isEmpty from 'lodash/isEmpty';
+import isEqual from 'lodash/isEqual';
+import omit from 'lodash/omit';
+import pick from 'lodash/pick';
+import remove from 'lodash/remove';
+import { setEndpoint } from '../../helpers/mutations';
+import Vue from 'vue';
+
+const save = (state, user) => {
+  const { activeUsers } = state;
+  user.contexts.forEach(context => {
+    setUserActivity(Vue, activeUsers, user, context);
```

Comment:
Why not use `Vue.set` inside `setUserActivity` instead of passing it as an arg? 🤔